package.json

```diff
@@ -8,9 +8,10 @@
     "build": "npm run build-ts && npm run resources && npm run tslint",
     "serve": "node dist/src/server.js",
     "watch-node": "nodemon dist/src/server.js",
-    "test": "npm run build && mocha --exit --timeout 60000 -r ts-node/register test/**/**/*.ts",
+    "test": "npm run unit-test && npm run int-test",
```

Comment:
One thing I noticed, `.only` doesn't only run the block I specified when using make verify => npm run test. Not sure if this is causing it...not critical though.

Reply:
was this before or did you try it with this PR ?

Reply:
I don't see it happen with the dev branch code, only with this PR.

Reply:
yes test now is split into two different mocha commands you need to have only in each set (int vs unit)